content/handbook/engineering/architecture/design-documents/offline_direct_transfer_migrations/_index.md

```
@@ -19,190 +23,167 @@ stale: false
 
-- 継続的な同期: ダイレクト転送は一度限りの移行をサポートするべきですが、ソースが変更されるにつれてdiffを同期することはサポートしません。これは顧客から要望されている別の機会ですが、このアーキテクチャの焦点ではありません。この作業はそれを妨げないようにするべきですが、この機能はスコープ外です。
-- バックアップ/リストア: オフライン転送は「エクスポート」ファイルを作成しますが、これはデータバックアップソリューションや他の形式のディザスターリカバリーではありません。
+- 継続的な同期: ダイレクト転送は 1 回限りの移行をサポートしますが、ソースが変更されるたびに差分を同期することはサポートしません。これは顧客が要求したもう 1 つの機会ですが、このアーキテクチャの焦点ではありません。この作業はそれを妨げるべきではありませんが、この機能はスコープ外です。
+- バックアップ/リストア: オフライン転送は「エクスポート」ファイルと考えられるものを作成しますが、これはデータバックアップソリューションでも、その他の災害復旧の形式でもありません。
 
-### Nice-to-haves
+### あれば便利
 
-- 古いインスタンスへの回避策: 古いバージョンの GitLab はサポートされませんが、回避策が可能かもしれません。詳細については [互換性のある GitLab バージョン](#互換性のある-gitlab-バージョン) を参照してください。ダイレクト転送へのすべての変更と同様に、可能な限り古いバージョンを考慮する必要があります。
-- インポータープラットフォームのサポート: これは要件ではありませんが、アイデアとして浮上しています。ダイレクト転送のインポート側が以前にエクスポートされたデータをサポートする必要があるようになったため、エクスポートされたデータが特定のデータスキーマに適合している限り、あらゆるソースからのインポートを許可する機会があります。この機会をこの提案に基づいて開くことは理想的ですが、よりシンプルなオプションと比較して複雑さが大幅に増加するため、要件ではありません。
+- 古いインスタンスのワークアラウンド: GitLab の古いバージョンはサポートされませんが、ワークアラウンドが可能な場合があります。詳細については、[互換性のある GitLab バージョン](#compatible-gitlab-versions) を参照してください。ダイレクト転送へのすべての変更と同様に、可能な場合は古いバージョンを考慮する必要があります。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)